lib/determine-basal/determine-basal.js

```diff
@@ -326,7 +326,14 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
     }
 
     // min_bg of 90 -> threshold of 65, 100 -> 70 110 -> 75, and 130 -> 85
-    var threshold = min_bg - 0.5*(min_bg-40);
+
+    // But if profile.threshold_setting is over 65 increase the very minimum glucose threshold
+    var minimum_threshold = 65;
```

Comment:
Should this be 60? I think the lowest you can set `min_bg` to is `80`, and `80 - 0.5*(80-40) = 60`

Reply:
Changed to 60 now